test/y2network/sysconfig/connection_config_writers/ethernet_test.rb

```diff
@@ -48,12 +48,12 @@ def file_content(scr_root, file)
   let(:ip_configs) do
     [
       Y2Network::ConnectionConfig::IPConfig.new(
-        Y2Network::IPAddress.from_string("192.168.122.1/24"), id:        :default,
-                                                              broadcast: Y2Network::IPAddress.from_string("192.168.122.255")
+        Y2Network::IPAddress.from_string("192.168.122.1/24"),
+        id: "", broadcast: Y2Network::IPAddress.from_string("192.168.122.255")
```

Comment:
now I am a bit lost, default is "" or nil?

Reply:
I've decided to go for an empty string.